react/StoreContextProvider.js

```diff
@@ -40,28 +40,31 @@ class StoreContextProvider extends Component {
     window.dataLayer = window.dataLayer || []
 
     return (
-      <PixelProvider>
-        <DataLayerProvider value={{ dataLayer: window.dataLayer }}>
-          <GtmScripts gtmId={gtmId} />
-          <ExtensionPoint id="store/pixel" />
-          <PageViewPixel />
-          <Helmet>
-            <title>{titleTag}</title>
-            <meta name="description" content={metaTagDescription} />
-            <meta name="keywords" content={metaTagKeywords} />
-            <meta name="copyright" content={storeName} />
-            <meta name="author" content={storeName} />
-            <meta name="country" content={country} />
-            <meta name="language" content={locale} />
-            <meta name="currency" content={currency} />
-            <meta name="robots" content={metaTagRobots || META_ROBOTS} />
-            <meta httpEquiv="Content-Type" content={CONTENT_TYPE} />
-          </Helmet>
-          <OrderFormProvider>
-            <div className="vtex-store__template">{this.props.children}</div>
-          </OrderFormProvider>
-        </DataLayerProvider>
-      </PixelProvider>
+      <Fragment>
+        <ExtensionContainer id="store/__icons" />
```

Comment:
I think we don't need an ExtensionContainer, is just an ExtensionPoint `store/icons` and we add only one pack, one store will not have more than one pack, don't you think?

Reply:
I was thinking of the store using an icon pack from styleguide and then overriding only some of the icons, using a custom pack, that's why I used the extension container